### ISSUE TITLE
Tests failing on latest rakudo

### ISSUE THREAD

**Issue by AlexDaniel** [niner/Inline-Perl5#113]
```
===> Testing: Inline::Perl5:ver<0.32>:auth<github:niner>
Use of uninitialized value in string eq at t/lib/TestPerl5Package.pm line 19.
# Failed test at t/perl5package.t line 14
Use of uninitialized value in string eq at t/lib/TestPerl5Package.pm line 19.
# Failed test at t/perl5package.t line 22
# Looks like you failed 2 tests of 8
===> Testing [FAIL]: Inline::Perl5:ver<0.32>:auth<github:niner>
```

**Comment by zoffixznet** [comment]
Looking at the version number, that might've been addressed by the fixes since https://github.com/rakudo/rakudo/issues/1728

**Comment by niner** [comment]
Apparently this was a bug in MoarVM introduced in https://github.com/MoarVM/MoarVM/commit/04e84740bf240b3a108a14fa46cbee8c9706b9aa
I reverted that commit. Tests pass now for me.

**Comment by zoffixznet** [comment]
Could a new version be released? It seems the current latest is 0.32; the same as the OP in https://github.com/rakudo/rakudo/issues/1728 was using, so at least a couple of bugs are fixed.

**Comment by niner** [comment]
I just uploaded 0.33 to CPAN
